cmd/query/app/flags.go

```diff
@@ -44,6 +44,7 @@ const (
 	queryStaticFiles        = "query.static-files"
 	queryUIConfig           = "query.ui-config"
 	queryTokenPropagation   = "query.bearer-token-propagation"
+	queryTenantPropagation  = "query.tenant-propagation"
```

Comment:
Please see https://github.com/jaegertracing/jaeger/pull/3688/files, there is already flag defined for the tenancy, can we not reuse that?